public/assets/css/main.css

```diff
@@ -101,6 +101,39 @@ figcaption {
 	box-shadow: rgb(157, 18, 60) 0 0 .2em;
 }
 
+ul {
+	list-style-type: none;
```

Comment:
This is also defined in `footer ul`. You can remove the `footer ul` one as, defining it for `ul` is enough